dcos/package.py

```diff
@@ -1384,6 +1385,20 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_version(self, software_version):
+        """Returns the first package version for a given software version, or
```

Comment:
This should probably return the most recent version that supports this software version